RFC Temp/0005-proof-of-relay.md

```
@@ -0,0 +1,688 @@
+There is a structure called `channel` that MUST contain at least the following fields:
+
+1. `source`: public key of the source node (A in this case)
+2. `destination`: public key of the destination node (beneficiary, B in this case)
+3. `balance` : an unsigned 96-bit integer
+4. `ticket_index`: an unsigned 48-bit integer
+5. `channel_epoch`: an unsigned 24-bit non-zero integer
+6. `status`: one of the `ChannelStatus` values
+
+```
+Channel {
+	source: [u8; |P_A|],
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)